### PR TITLE
Add RTree2D

### PR DESCRIPTION
**Pull request by plokhotnyuk**

### CODE REVIEW

**Comment by krisskross** [comment]
Nice! Please fix merge first.

**Comment by plokhotnyuk** [comment]
@krisskross what is wrong with the submitted pull request?

**Comment by izeye** [comment]
@plokhotnyuk I'm not sure but I guess @krisskross requested you to fix the Travis CI build failure. You can see the reason by clicking its "Details" button. However, the failure looks irrelevant with this PR and caused by the other entries AFAICT:

```
Issues :-(
> Links 
  1. [L067]  http://j-php.net getaddrinfo: Name or service not known 
  2. [L332]  https://robovm.com/ getaddrinfo: No address associated with hostname 
```

**Comment by plokhotnyuk** [comment]
@izeye @krisskross, thank you both! 

I have updated broken links.

File: readme.md [hunk 1]
```diff
@@ -328,7 +329,7 @@ A curated list of awesome JVM low level, performance and non-framework related s
 * [J2ObjC](https://github.com/google/j2objc) - Translator from Java source to Objective-C code. Keeps shared code between iOS native apps and Android native apps. 
 * [jvm.go](https://github.com/zxh0/jvm.go) - A JVM written in Go.
 * [ParparVM](https://github.com/codenameone/CodenameOne/tree/master/vm) - An Open Source Java bytecode to C translator for iOS native development. Designed as a part of the [Codename One](https://www.codenameone.com/) WORA for mobile project.
-* <strike>[RoboVM](https://robovm.com/) - Create native iOS and Android apps in Java.</strike> - Discontinued
+* [MobiDevelop's RoboVM Fork](https://github.com/MobiVM/robovm) - Ahead of time compiler for JVM bytecode targetting iOS, Mac OSX and Linux
```
Comment (izeye) [inline readme.md]: A typo for "targeting" and a missing closing period.

**Pull request merged**